internal/cmd/dev_sqld_version.go

```diff
@@ -0,0 +1,24 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+func getSqldVersion(cmd *cobra.Command) {
+	cmd.Flags().BoolP("version", "v", false, "sqld version")
```

Comment:
Please bind the flag to a variable the same way it's done in https://github.com/tursodatabase/turso-cli/blob/main/internal/cmd/dev_port_flag.go and use that variable instead of calling `cmd.Flags().GetBool("version")`